var/spack/repos/builtin/packages/py-pyqt5/package.py

```diff
@@ -12,45 +12,30 @@ class PyPyqt5(SIPPackage):
     Windows, OS X, Linux, iOS and Android. PyQt5 supports Qt v5."""
 
     homepage = "https://www.riverbankcomputing.com/software/pyqt/intro"
-    url = (
-        "https://www.riverbankcomputing.com/static/Downloads/PyQt5/5.13.0/PyQt5_gpl-5.13.0.tar.gz"
```

Comment:
This URL doesn't work and all older versions are unavailable, so I deprecated them

Reply:
I remember noticing this issue a while ago. If necessary, we might be able to use internet archive's wayback machine to get older tars.

Reply:
This is why Spack should re-mirror a lot of these things. With content-addressed lookup, the URL can just be `archive.spack.tld/archives/$algo/$hash`.

Reply:
Spack does re-mirror all URLs. On develop:
```console
$ spack fetch py-pyqt5
==> Fetching https://mirror.spack.io/_source-cache/archive/54/54b7f456341b89eeb3930e786837762ea67f235e886512496c4152ebe106d4af.tar.gz
```
However, I don't think anyone really cares about installing these older versions, and newer versions are available on PyPI, where releases actually can't be removed, only yanked. So this new URL should be more reliable for folks not using Spack's mirror.

Reply:
Releases can definitely be deleted from PyPI; I've had to do it for VTK multiple times because we hit storage quotas.

Reply:
Ah, maybe it's that you can't rerelease the same version

Reply:
Yes, once a release is deleted on PyPI, it is gone forever.